lib/ioki/apis/platform_api.rb

```diff
@@ -167,6 +167,16 @@ class PlatformApi
         actions:     { 'lock' => :patch, 'unlock' => :patch, 'set_pin' => :patch },
         path:        [API_BASE_PATH, 'products', :id, 'drivers', :id],
         model_class: Ioki::Model::Platform::Driver
+      ),
+      Endpoints::Index.new(
```

Comment:
We have the `crud_enpoints` "macro" for stuff like this:
```ruby
Endpoints.crud_endpoints(
  :announcement,
  base_path:   [API_BASE_PATH, 'products', :id],
  except: [:create, :update, :delete],
  model_class: Ioki::Model::Platform::Announcement
)
```

Reply:
Thanks, done :)